_notices/rsn0047.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 47 # should match notice number
+notice_pin: true # set to true to pin to notice page
+
+title: "Changes to RAPIDS' branching strategy in 25.08"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Development Process Change
+notice_rapids_version: "v25.08+"
+notice_created: 2025-05-05
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2025-05-05
+---
+
+## Overview
+
+RAPIDS will be migrating to a new branching strategy for the 25.08 release. The new 
+strategy will use a single long-lived branch called `main`, which all pull requests 
+should target. In preparation for a release, a release branch will be created from 
+`main` and any remaining pull requests for that release should be merged into the
+release branch.
+
+Releases and hot fixes will be tagged from the release branch.
+
+This new strategy will be piloted on several RAPIDS libraries (`rapidsai/rmm`, 
+`rapidsai/kvikio`, and `rapidsai/cugraph-gnn`) during the `25.08` development cycle.
+Migrations will begin shortly after the 25.06 release. After the `25.08` release,
+the new strategy will be implemented across all RAPIDS libraries.
+
+## Impact
+
+Below is a table comparing the current and new branching strategies. Here, the current release is `YY.MM` (e.g. 25.06) and we denote the next release as `<YY.MM-next>` (e.g. 25.08).
+
+Aspect | Current (25.06) | New Branching Strategy (25.08+)
+-- | -- | --
+Default Branch | Switched per release (e.g. `branch-YY.MM`, then `branch-<YY.MM-next>`) | Always `main` (latest and greatest)
+Feature PRs | Target current default branch; retarget manually if moving to a different release | Always target `main`. During release preparation, pull requests may need to be re-targeted to the release branch.
+Release Branching | No specific branch, `branch-YY.MM` is the release branch | Create `release/YY.MM` for releases, used during burndown & code freeze
+Burndown Process | `branch-<YY.MM-next>` is created | Bump `main` to `<YY.MM-next>.00`, tag `v<YY.MM-next>.00a`, create branch `release/YY.MM`, enable forward merges from `release/YY.MM` → `main`
+Code Freeze | Only admin-merges to `branch-YY.MM` | Only admin-merges to `release/YY.MM`
+Release | (What to say here?) | Merge `release/YY.MM` → `main`; disable forward-merges from `release/YY.MM` → `main`; tag `vYY.MM.00` on `release/YY.MM`

```

Comment:
@raydouglass Please take a look over this table. This line is the only one where I wasn't sure what to say for the "Current" strategy.